projects/lcms/cms_gdb_fuzzer.c

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2023 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <stdint.h>
+#include "lcms2.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+
+    if (size < 16) {
+        return 0;
+    }
+    //cmsGBDAlloc
+    cmsHANDLE hGDB = cmsGBDAlloc(NULL);
+    if (!hGDB){
+        return 0;
+    }
+    //cmsGDBAddPoint
+    cmsCIELab Lab;
+    Lab.L = *((const uint32_t *)data);
+    Lab.a = *((const uint32_t *)data+1);
+    Lab.b = *((const uint32_t *)data+2);
+    cmsGDBAddPoint(hGDB, &Lab);
+    
+    //cmsGDBCheckPoint
+    cmsGDBCheckPoint(hGDB, &Lab);
+
+    //cmsGDBCompute
+    cmsGDBCompute(hGDB, *((const uint32_t *)data+3));
```

Comment:
This seems like a narrow fuzzer -- it has a total of 16 bytes of data to be mutated.

Reply:
Yes, we just have 4 places to put data.